### PR TITLE
Add www as a CNAME for the admin platform

### PR DESCRIPTION
**Pull request by emileswarts**
This also requires setting up a new SSL certificate for the www domain.
There is another task here to redirect all requests to the www version
of the site.  This will be picked up as an isolated task as we need to
install and configure Nginx.

### CODE REVIEW

File: govwifi-admin/route53.tf [hunk 1]
```diff
@@ -9,6 +9,22 @@ resource "aws_route53_record" "admin" {
...
+  ttl     = "300"
+  records = ["admin.${var.Env-Subdomain}.service.gov.uk"]
+}
+
+resource "aws_route53_record" "docs_www" {
+  zone_id = "${data.aws_route53_zone.zone.id}"
+  name    = "www.docs.${var.Env-Subdomain}.service.gov.uk"
+  type    = "CNAME"
+  ttl     = "300"
+  records = ["docs.${var.Env-Subdomain}.service.gov.uk"]
```
Comment (cybojenix) [inline govwifi-admin/route53.tf]: Where is this record configured?

Comment (emileswarts) [reply govwifi-admin/route53.tf]: Route53

Comment (emileswarts) [reply govwifi-admin/route53.tf]: Or do you mean where in Terraform?
I'm not sure it looks like it was set up outside of Terraform.

**Comment by emileswarts** [comment]
All our wifi.service.gov.uk dns is managed in route53 @kjdchapman

**Pull request merged**